### PR TITLE
Follow-up on #1068

### PR DESCRIPTION
**Pull request by serban300**
Addresses some comments from #1068

Also defines `StorageMapKeyProvider`

### CODE REVIEW

File: primitives/runtime/src/lib.rs [hunk 1]
```diff
@@ -285,6 +285,28 @@ pub fn storage_value_key(pallet_prefix: &str, value_name: &str) -> StorageKey {
 	StorageKey(final_key)
 }
 
+/// Can be use to access the runtime storage key of a `StorageDoubleMap`.
```
Comment (svyatonik) [inline primitives/runtime/src/lib.rs:288]: Iiuc it isn't about `StorageDoubleMap`, but about `StorageMap`

File: primitives/runtime/src/lib.rs [hunk 2]
```diff
@@ -285,6 +285,28 @@ pub fn storage_value_key(pallet_prefix: &str, value_name: &str) -> StorageKey {
...
+pub trait StorageMapKeyProvider {
+	// The name of the variable that holds the `StorageDoubleMap`
```
Comment (svyatonik) [inline primitives/runtime/src/lib.rs:290]: Let's change it to `///` maybe (and below as well)

**Review by svyatonik — APPROVED** [review]
LGTM

File: primitives/runtime/src/lib.rs (hunk 1 above)
Comment (serban300) [reply primitives/runtime/src/lib.rs]: True. Done.

File: primitives/runtime/src/lib.rs (hunk 2 above)
Comment (serban300) [reply primitives/runtime/src/lib.rs]: Done.

**Pull request merged**